src/razor/src/semantic/provideSemanticTokensResponse.ts

```diff
@@ -3,9 +3,7 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { SemanticTokensResponse } from './semanticTokensResponse';
-
 export class ProvideSemanticTokensResponse {
     // tslint:disable-next-line: variable-name
-    constructor(public Result: SemanticTokensResponse, public HostDocumentSyncVersion: number | null) {}
+    constructor(public Tokens: Array<Array<number>>, public HostDocumentSyncVersion: number | null) {}
```

Comment:
The [server version](https://github.com/dotnet/razor/blob/main/src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Models/ProvideSemanticTokensResponse.cs) does not have `HostDocumentSyncVersion` as nullable, so could be causing an exception during deserialization.